nixos/modules/programs/mouse-actions.nix

```diff
@@ -3,13 +3,39 @@
 let
   cfg = config.programs.mouse-actions;
 in
-  {
-    options.programs.mouse-actions = {
-      enable = lib.mkEnableOption ''
-        mouse-actions udev rules. This is a prerequisite for using mouse-actions without being root.
+{
+  options.programs.mouse-actions = {
+    enable = lib.mkEnableOption ''
+      mouse-actions udev rules. This is a prerequisite for using mouse-actions without being root.
+    '';
+    package = lib.mkOption {
+      type = lib.types.package;
+      example = lib.literalExpression "pkgs.mouse-actions-gui";
+      default = pkgs.mouse-actions;
+      defaultText = lib.literalExpression "pkgs.mouse-actions";
+      description = "Which mouse-actions package to use.";
+    };
```

Comment:
```suggestion
    package = lib.mkPackageOption pkgs "mouse-actions" {
      example = "pkgs.mouse-actions-gui";
    };
```

Reply:
Great, didn't know about this. I don't see other packages using the `pkgs.` prefix when using this so I'll omit it too.